cli/validate.ts

```diff
@@ -26,10 +27,13 @@ export async function handleValidate({manifestFile}: ArgumentsCamelCase<Validate
   }
 
   const metadata = compilePackMetadata(manifest);
-  return validateMetadata(metadata);
+  return validateMetadata(metadata, {checkDeprecationWarnings});
 }
 
-export async function validateMetadata(metadata: PackVersionMetadata) {
+export async function validateMetadata(
+  metadata: PackVersionMetadata,
+  {checkDeprecationWarnings = true}: {checkDeprecationWarnings?: boolean} = {},
```

Comment:
Adding the ability to turn this off when called directly, since the `packs` repo will likely have thousands of such warnings before we go fix everything.